apps/skilavottord/ws/src/app/modules/auth/user.model.ts

```diff
@@ -19,4 +19,7 @@ export class User {
 
   @Field(() => Role)
   role!: Role
+
+  @Field({ nullable: true })
```

Comment:
Let's remove the field ... but keep the property. Don't need to expose this to GraphQL